cla_frontend/assets-src/javascripts/app/partials/call_centre/case_detail.assign.html

```diff
@@ -48,27 +48,29 @@ <h2 class="FormBlock-label">Matter types</h2>
-          <p>If you qualify for legal aid you will need to provide proof of your financial circumstances to the specialist/legally aided Education provider.</p>
-          <p>If they do take on your case they will provide you legal advice remotely, via email, letters and phone calls until the conclusion of your case. You won’t be able to get advice from any other legal aid providers on this issue.</p>
-          <strong>Thursday and Friday</strong>
-          <p>Based on what you have told me I can put you through to a Legally Aided Education Provider. They will check what you have told me and assess whether our service is right for you.</p>
-          <p>If you qualify for legal aid you will need to provide proof of your financial circumstances to the specialist/legally aided Education provider.</p>
-          <p>If they do take on your case they will provide you legal advice remotely, via email, letters and phone calls until the conclusion of your case. You won’t be able to get advice from any other legal aid providers on this issue.</p>
-          <strong>Saturday</strong>
+          <strong>Monday to Thursday</strong>
+          <p>Based on what you have told me today I can put you through to a Civil Legal Advice Specialist Advisor, they will check what you have told me and assess whether our service is right for you.</p>
+          <p>If you qualify for legal aid you will need to provide proof of your financial circumstances to the specialist.</p>
+          <p>If they do take on your case they will provide you legal advice remotely via email, letters and phone calls until the conclusion of your case. You won’t be able to get advice from any other legal aid providers on this issue.</p>
+          <p>Should you prefer face to face to remote advice, I can instead provide you three face to face providers in your area who you can contact for assistance.</p>
+          <p>(If the client chooses remote assistance: ) Is it ok to transfer you to a civil legal advice provider?</p>
+          <p>Is there anything else we can help you with today?</p>
+          <p>If you need further help please call us back. We are open 9am – 8pm Monday to Friday and 9am-12:30pm on Saturdays.</p>
+          <strong>Friday and Saturday</strong>
           <p>Based on what you have told me I can provide you with details of legally aided Education providers in your area. They will check what you have told me and assess whether our service is right for you.</p>
           <strong>All days</strong>
           <p>Is there anything else we can help you with today?</p>
-          <p>If you need any further help please call us back. We are open from 9am-8pm Monday to Friday and 9am – 12:30pm on Saturdays.</p>
+          <p>If you need further help please call us back. We are open 9am – 8pm Monday to Friday and 9am-12:30pm on Saturdays.</p>
```

Comment:
weird mix of hyphens is this what they wanted?

Reply:
As per Rez's updates https://dsdmoj.atlassian.net/browse/LGA-3159 yep